src/main/java/com/angelbroking/smartapi/http/exceptions/CustomException.java

```diff
@@ -0,0 +1,20 @@
+package com.angelbroking.smartapi.http.exceptions;
+
+public class CustomException extends RuntimeException {
```

Comment:
Please rename to a better class name.

Reply:
have been renamed to SmartConnectException